README.md

```diff
@@ -104,11 +107,37 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 
 #### GitHub permission scopes
 
+##### Repository Permissions
+
+|**Permission scope**|**Why the app needs it**|
+|---|---|
+|**Read** access to contents & metadata |**Contents** (aka code) and **metadata** are needed to sync development information to Jira. <br><br> **Medadata:** All GitHub apps have read-only metadata permission set by default. This is a [mandatory requirement by GitHub](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions) and is needed to provide access to a collection of read-only endpoints with metadata for various resources. These endpoints do not provide sensitive private repository information. Read-only metadata permissions are used for the following webhook: <br> - repository (all events excluding `deleted`) <br><br> **Contents:** Read-only content permissions are used for the following webhooks: <br> - commit comment <br> - create <br> - delete <br> - push <br> - workflow run|
```

Comment:
why are we using `<br>`?  We don't need HTML in markdown...

Reply:
Without the break tags, the content looks like this:

<img width="1123" alt="Screen Shot 2022-01-18 at 9 20 34 am" src="https://user-images.githubusercontent.com/37155488/149843002-baeffb5a-65a7-4d25-9d7b-adc9b0f17e69.png">

Vs this:

<img width="1100" alt="Screen Shot 2022-01-18 at 9 21 53 am" src="https://user-images.githubusercontent.com/37155488/149843091-bbb30363-1424-40b7-be7c-14c2c24d492f.png">